src/Core/Utils/Attribs.hpp

```diff
@@ -19,58 +19,109 @@ namespace Utils {
 template <typename T>
 class Attrib;
 
-/// AttribBase is the base class for attributes of all type.
+/**
+ * AttribBase is the base class for attributes of all type.
+ */
 class AttribBase {
   public:
     explicit AttribBase( const std::string& name ) : m_name{name} {}
     virtual ~AttribBase() {}
+
+    /**
+     * Return the attribute's name.
+     */
     std::string getName() const { return m_name; }
+
+    /**
+     * Set the attribute's name.
+     */
     void setName( const std::string& name ) { m_name = name; }
+
+    /**
+     * Resize the attribute's content.
+     */
     virtual void resize( size_t s ) = 0;
 
-    virtual uint getSize() = 0;
+    /**
+     * Return the number of elements in the attribute content.
+     */
+    virtual size_t getSize() = 0;
+
+    /**
+     * Return the size, in bytes, of one attribute element.
```

Comment:
Well stide is the offset, in byte, to go from one attirbute address to ne next one.
As defined here https://www.khronos.org/opengl/wiki/Vertex_Specification